src/index.js

```diff
@@ -39,9 +40,12 @@ export default function supernova(env) {
       const constraints = useConstraints();
       const translator = useTranslator();
       const selectionsAPI = useSelections();
-      const tableTheme = muiSetup(direction);
-      // the style which is from them can be found in nebula.config.js -> serve -> themes
-      const theme = useTheme();
+      const theme = {
+        ...useTheme(),
```

Comment:
would this update properly?

Reply:
I think so

Reply:
For consistency and robustness I'd change `theme` after you run useTheme

```suggestion
const theme = useTheme(),
theme.backgroundColor = ....
```

Reply:
Yeah, sure